### ISSUE TITLE
[kaeld on 2007-11-26 20:36:47] : Flasher channels not all showing up in the data stream

### ISSUE THREAD

**Issue by dglo** [dglo/pdaq#647]
No associated GitHub commit

**Comment by dglo** [comment]
[kaeld on 2007-11-26 22:31:55]
Here is the error - the DOM in question is 'Mulberry' - could the flasherboard be bad (??)

stringHub icecube.daq.stringhub.StringHubComponent-XML-RPC-0 INFO [2007-11-08 18:20:45.613] Beginning subrun
stringHub icecube.daq.domapp.DataCollector-01B INFO [2007-11-08 18:20:45.855] Starting flasher subrun
stringHub icecube.daq.domapp.DataCollector-01B INFO [2007-11-08 18:20:45.855] Configuring DOM on [01B]
stringHub icecube.daq.domapp.DataCollector-00B INFO [2007-11-08 18:20:45.855] Starting flasher subrun
stringHub icecube.daq.domapp.DataCollector-00B INFO [2007-11-08 18:20:45.855] Configuring DOM on [00B]
stringHub icecube.daq.domapp.DataCollector-01B INFO [2007-11-08 18:20:48.061] Finished DOM configuration - [01B]; configuration took 2206 milliseconds.
stringHub icecube.daq.domapp.DataCollector-00B INFO [2007-11-08 18:20:48.070] Finished DOM configuration - [00B]; configuration took 2215 milliseconds.
stringHub STDERR-01B ERROR [2007-11-08 18:20:48.342] DOM Message exception: sent type = BEGIN_FB_RUN, returned type = 4, returned subtype = 27, status = 11111111111111111111111111000000b
stringHub STDERR-01B ERROR [2007-11-08 18:20:48.342] 	at icecube.daq.domapp.DOMApp.sendMessage(DOMApp.java:341)
stringHub STDERR-01B ERROR [2007-11-08 18:20:48.342] 	at icecube.daq.domapp.DOMApp.beginFlasherRun(DOMApp.java:42)
stringHub STDERR-01B ERROR [2007-11-08 18:20:48.342] 	at icecube.daq.domapp.DataCollector.runcore(DataCollector.java:936)
stringHub STDERR-01B ERROR [2007-11-08 18:20:48.342] 	at icecube.daq.domapp.DataCollector.run(DataCollector.java:686)

**Comment by dglo** [comment]
[kaeld on 2007-11-27 20:21:15]
Mark Krasberg likely has tracked the problem to an old FB firmware on the DOM ...

Kael

I checked "Mulberry" on SPTS. It failed the flasherboard STF tests.
I checked and it had old flasherboard firmware on it.

I have just loaded the latest firmware onto it, using the special procedure to try to resurrect older flasherboards. This worked and the DOM
now passes the STF flasher tests.

thanks
Mark
